tests/Directory.Build.props

```diff
@@ -19,6 +19,7 @@
 
   <ItemGroup>
     <PackageReference Include="GitHubActionsTestLogger" PrivateAssets="all" IncludeAssets="runtime; build; native; contentfiles; analyzers" />
+    <Using Include="Xunit.Abstractions" />
```

Comment:
Please place it in own ItemGroup to increase maintainability

Reply:
Done